### PR TITLE
Bugfix #51 HaaS to HIL exception

### PR DESCRIPTION
**Pull request by xukan**
[https://github.com/CCI-MOC/ims/issues/51](url)

### CODE REVIEW

**Comment by chemistry-sourabh** [comment]
@xukan you are comparing it with master not dev and I told you to squash the commits into one

**Comment by chemistry-sourabh** [comment]
@xukan I squashed your commits with into one commit. You will need to change this PR to compare with dev not master and fix your failed check.

Also check whether all your changes are there

**Comment by chemistry-sourabh** [comment]
Cool looks like you got it right this time :)

File: ims/cli/cli.py [hunk 1]
```diff
@@ -272,6 +272,7 @@ def list_projects():
     \b
     WARNING = User Must be An Admin
     """
+    print("First try")
```
Comment (chemistry-sourabh) [inline ims/cli/cli.py]: Remove this

**Comment by chemistry-sourabh** [comment]
@xukan  looks like you removed all HaaSes, so fix that 1 nit and I will approve it.

Also @naved001 safe side just test this also.

**Comment by ravisantoshgudimetla** [comment]
Wondering, if we can add copy right terms here to all the files now that we are touching too many of them. @chemistry-sourabh, can you guide @xukan on getting that information from HIL and adding them here.

**Comment by chemistry-sourabh** [comment]
@ravisantoshgudimetla Let the copyrights thing be in another PR. We should not deviate from our minimum PR rule. If it was me you guys would have wanted a PR for each file :P.

Also I dont know the copyright thing in HIL, perhaps @naved001 can point it out or copy paste in the copyright issue for @xukan to use.

Also @xukan rebase your PR and try to be more active on github.

File: .gitignore [hunk 2]
```diff
@@ -85,6 +85,7 @@ celerybeat-schedule
 # virtualenv
 venv/
 ENV/
+.venvwrapper/
```
Comment (chemistry-sourabh) [inline .gitignore]: Did you commit it by mistake? Why don't you change the name of your .venvwrapper to venv? or you can do a local gitignore.

**Comment by chemistry-sourabh** [comment]
@CCI-MOC/bmi-breakers  someone else can review this PR. I am done

**Comment by chemistry-sourabh** [comment]
Ok I approve this @CCI-MOC/bmi-breakers someone else can review for second +1

**Comment by chemistry-sourabh** [comment]
@xukan I made another observation, why did you merge CCI-MOC/dev into your PR instead of rebasing it ? Merging master/dev into your PR effects history. Please undo it

**Pull request closed without merging**